web/ui/dashboard-component/projects/convoy-dashboard/src/lib/convoy-dashboard.component.html

```diff
@@ -183,16 +188,17 @@ <h3>Configuration</h3>
 						</button>
 					</li>
 					<div class="color__grey font__14px font__weight-500 font--nowrap margin-left--auto padding-right__14px">
-						<span [hidden]="activeTab !== 'events'">
+						<span *ngIf="events" [hidden]="activeTab !== 'events'">
```

Comment:
why did you add *ngIf when there's [hidden]

Reply:
I added it because I only want to show the pagination data if events data exists, else it'll throw errors. @Oluwadaminiola